.circleci/config.yml

```diff
@@ -1,49 +1,69 @@
 version: 2
 jobs:
+  vault-token:
+    docker:
+      - image: circleci/buildpack-deps:curl
+    environment:
+      VAULT_ADDR: https://vault.afdevops.com
+    working_directory: /tmp
```

Comment:
@dmitriy-lukyanchikov what will happen if we will keep default `working_directory`?

Reply:
i think it will work in the same way but did not test it